testing/python_lib/test_fot.py

```diff
@@ -135,5 +137,51 @@ def test_receiving_device_testing_states(self):
         self.assertEqual(sorted_received_states, sorted_expected_states)
 
 
+class FOTPortsTestingStatesTestCase(PortsTestingStateManagerTestBase):
```

Comment:
And this is exactly why we shouldn't call it "TestingStates" -- because Test is used twice with different meanings.

Reply:
Changed to "FotPortStatesTestCase".